docs/modules/ROOT/pages/Technical Documentation/Payara Micro Documentation/Payara Micro Configuration and Management/Micro Management/Configuring An Instance.adoc

```diff
@@ -0,0 +1,374 @@
+[[configuring-an-instance]]
+= Configuring an Instance
+
+This section details how to configure a Payara Micro instance.
+
+[[configuring-an-instance-from-the-command-line]]
+== Configuring an Instance from the Command Line
+
+As described in
+xref:/Technical Documentation/Payara Micro Documentation/Payara Micro Configuration and Management/Micro Management/Deploying Applications/deploy-cmd-line.adoc[Deploying From the Command Line],
+the starting and configuration of an instance can be done in its entirety on
+one line.
+
+The options available can be seen by running the JAR with the `--help` option,
+or by consulting the
+xref:/Technical Documentation/Payara Micro Documentation/Payara Micro Configuration and Management/Micro Management/Command Line Options/Command Line Options.adoc[Payara Micro Command Line Options]
+section.
+
+The general structure of starting, configuring, and deploying an application to
+an instance is as follows:
+
+[source, java]
+----
+java -jar payara-micro.jar _--option1_ _--option2_ ...
+----
+
+As an example, see below for starting an instance with a non-default HTTP port:
+
+[source, java]
+----
+java -jar payara-micro.jar --port 2468
+----
+
+[[warmup]]
+=== Separating Configuration from Production Run
+
+An instance can be configured separately, but only when xref:/Technical Documentation/Payara Micro Documentation/Payara Micro Configuration and Management/Micro Management/Jar Structure & Configuration/rootdir.adoc[persistent root configuration directory] is specified by means of command line argument `--rootDir`.
```

Comment:
Just a FYI this link is broken, but its not too big of a deal since the page it links too needs to be merged at some point